test/cli/build.js

```diff
@@ -93,7 +93,10 @@ test('`fusion build` works in production with a CDN_URL', async t => {
     'Server Entry file sourcemap gets compiled'
   );
   const {res, proc} = await start(`--dir=${dir}`, {
-    env: Object.assign({}, process.env, {CDN_URL: 'https://cdn.com/test'}),
+    env: Object.assign({}, process.env, {
```

Comment:
Can we also make a new test case instead of updating this one? Would be nice to be sure that we have a test case for only the new functionality, while preserving coverage for older cases.

Reply:
yes. added new one